assets/stylesheets/main.css

```diff
@@ -354,8 +406,8 @@ a:hover#join {
   align-items: center;
   justify-content: space-between;
   flex-wrap: wrap;
-  height:92vh;
-  width: 90%;
+  height:calc(100vh - 16vh);
```

Comment:
height of window needs to incorporate the height of the WCG header (8vh) and the height of the subheader (8vh)